test/Xunit.StaFact.Tests/StaFactTests.cs

```diff
@@ -11,7 +11,7 @@ public StaFactTests()
         Assert.Equal(ApartmentState.STA, Thread.CurrentThread.GetApartmentState());
     }
 
-    [StaFact]
+    [DesktopFact]
```

Comment:
ℹ️ Some of these tests were using `StaFact` and others were using `DesktopFact` (which is an alias for `StaFact`. I changed them to all use `DesktopFact`. I did the same thing in the other test files.